app/main.ipynb

```diff
@@ -240,24 +225,29 @@
     "Get the max hours per week some person works. How many people works the same amount of hours per week?"
    ],
    "metadata": {
-    "collapsed": false,
-    "pycharm": {
-     "name": "#%% md\n"
-    }
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 90,
+   "outputs": [],
+   "source": [
+    "print(\"max_hours_per_week =\", df[\"hours-per-week\"].max(), \"hours\")"
+   ],
+   "metadata": {
+    "collapsed": false
    }
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 91,
    "outputs": [],
    "source": [
-    "# write your code for task 8 here"
+    "print(df[[\"hours-per-week\", \"sex\"]].groupby(by=\"hours-per-week\").count())"
```

Comment:
Better to create alias for second column, count_of_people or smth like this, so it will be clear what column really shows
![image](https://user-images.githubusercontent.com/44442087/221360090-df653255-3983-413e-8848-e7de43e74043.png)